modules/dataquery/jsx/react.tabs.js

```diff
@@ -213,7 +213,7 @@ var ViewDataTabPane = React.createClass({
         window.dataBlobs = [];
 
         if(FileList.length < 100 || confirm("You are trying to download more than 100 files. This may be slow or crash your web browser.\n\nYou may want to consider splitting your query into more, smaller queries by defining more restrictive filters.\n\nPress OK to continue with attempting to download current files or cancel to abort." )) {
-            saveworker = new Worker(loris.BaseURL + '/GetJS.php?Module=dataquery&file=workers/savezip.js');
+            saveworker = new Worker(loris.BaseURL + '/dataquery/js/workers/savezip.js');
```

Comment:
why is the compiled file above (can't comment on it) still part of this branch? Shouldn't it be generated by make?

Reply:
This is going to bugfix on version 20.3.0, I'd rather have it in v20.3.1 given the difficulty for projects to go to 21 and the need for this to use the DQT